python/trifinger_simulation/sim_finger.py

```diff
@@ -18,6 +18,16 @@ class SimFinger:
     A simulation environment for the single and the tri-finger robots.
     This environment is based on PyBullet, the official Python wrapper around
     the Bullet-C API.
+
+    Attributes:
+        finger_type (string): Name of the finger type.  Use
+                :meth:`get_valid_finger_types` to get a list of all supported
+                types.
+        time_step (float): Time (in seconds) between two simulation steps.
+            Don't set this to be larger than 1/60.  The gains etc. are set
+            according to a time_step of 0.004 s.
+        enable_visualization (bool): Set this to 'True' for a GUI interface
+            to the simulation.
```

Comment:
I would rather add this in the code below, using `#:` above the atrributes.  Then there is less distance between code and documentation which makes it less likely to get out of sync.

Reply:
Ok, I added it like this to be similar to the way it is in Action and Observation. Can we keep it like this for now, and then change all of these consistently later?

Reply:
sure, this is not critical